TrackletAlgorithm/MatchEngineUnit.h

```diff
@@ -256,7 +262,7 @@ inline void step(const VMStubMECM<VMSMEType> stubmem[4][1024]) {
 inline bool processing() {
 #pragma HLS inline  
   return !idle_||good__||good__t||good___;
```

Comment:
@jasonfan393 Please indent these lines to be consistent with where the closing curly brace is now.